examples/stable-diffusion/training/train_text_to_image_sdxl.py

```diff
@@ -1547,102 +1440,79 @@ def compute_time_ids(original_size, crops_coords_top_left):
 
                 del pipeline
 
-    duration = time.perf_counter() - t0
-    throughput = args.max_train_steps * total_batch_size / duration
+    duration = time.perf_counter() - t0 - (checkpoint_time if args.adjust_throughput else 0)
+    ttt = time.perf_counter() - t_start
+    throughput = (args.max_train_steps - args.throughput_warmup_steps) * total_batch_size / duration
 
     accelerator.wait_for_everyone()
     if accelerator.is_main_process:
         logger.info(f"Throughput = {throughput} samples/s")
         logger.info(f"Train runtime = {duration} seconds")
+        logger.info(f"Total Train runtime = {ttt} seconds")
```

Comment:
Not very clear, I would just keep the TTT

Reply:
As we had long graph compilation for R1.15 as known issue, if we keep just TTT, we don't show the train time excluding graph compilation time. We will remove the train runtime after the graph compilation issue is fixed in R1.16. WDYT?